apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -361,6 +374,21 @@ class ChromebookInstructions extends React.Component {
       </div>
     );
   }
+
+  render() {
+    let setupInstructions;
+    if (experiments.isEnabled('webserial')) {
+      setupInstructions = this.webSerialSetupInstructions();
+    } else {
+      setupInstructions = this.chromeAppSetupInstructions();
+    }
```

Comment:
super nit: i think a ternary would be more readable here. honestly, you could even skip setting the variable and instead conditionally render this in JSX. not sure which i like better, but either seems like a good option!